app/controllers/application_controller.rb

```diff
@@ -1,10 +1,15 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :create_cart
```

Comment:
@tochman This specific case is not performance heavy as I pointed out in my previous comment so I'm okay with letting this go. I just thought it was necessary to point out the implications of doing something like this.